sup3r/models/multi_step.py

```diff
@@ -320,6 +374,57 @@ def load(cls, spatial_model_dirs, temporal_model_dirs, verbose=True):
 
         return cls(s_models, t_models)
 
+    def _split_exo_dict(self, split_step, exogenous_data=None):
```

Comment:
I haven't looked far enough to know the answer to this, but shouldn't you not have to split the exo_data dict into spatial/temporal parts now that each entry has an explicitly defined model step? Like wouldn't the extra model steps just be passed through and unused?

Reply:
Hmmm yeah we talked about this right? Getting rid of the spatial+temporal model class? Sounding like a better and better idea! Flat is better than nested, right?

Reply:
Yep! Removed the spatial+temporal gan models but kept the Base class and changed the Solar model to subclass that base class.